src/docs/development/add-to-app/ios/add-flutter-screen.md

```diff
@@ -564,7 +559,7 @@ Waiting for a connection from Flutter...
 Done.
 Syncing files to device...      1.1s
 
-🔥  To hot reload changes while running, press "r". To hot restart (and rebuild state), press "R".
+To hot reload changes while running, press "r". To hot restart (and rebuild state), press "R".
```

Comment:
This is a specific quoting of a terminal output from the tools

Reply:
I sort of figured that, but I thought I’d try anyway.